### PR TITLE
add text to README

### PR DESCRIPTION
**Pull request by clairefields15**
Added some cool stuff. check it out

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
 # Knitting is Witchcraft
+
+
+It totally is
```
Comment (natalia2103) [inline README.md:4]: We are in agreement. Nice.

File: README.md [hunk 2]
```diff
@@ -1 +1,8 @@
...
+
+Not crafting doesn't make you a bad person
```
Comment (natalia2103) [inline README.md:6]: But doesn't it??

File: README.md [hunk 3]
```diff
@@ -1 +1,8 @@
...
+
+Crocheting is weird.
```
Comment (natalia2103) [inline README.md]: Can we change "weird" to "special"?

**Review by natalia2103 — CHANGES_REQUESTED** [review]
I left some feedback!!

File: README.md (hunk 3 above)
Comment (clairefields15) [reply README.md]: Got it!

Comment (natalia2103) [reply README.md]: Great Job!!

**Pull request merged**